integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/filterexpr/CountStarTestCase.scala

```diff
@@ -65,6 +65,51 @@ class CountStarTestCase extends QueryTest with BeforeAndAfterAll {
         CarbonCommonConstants.ENABLE_QUERY_STATISTICS_DEFAULT)
   }
 
+  test("select query without filter should not be pruned with multi thread") {
+    val numOfThreadsForPruning = CarbonProperties.getNumOfThreadsForPruning
+    val carbonDriverPruningMultiThreadEnableFilesCount =
+      CarbonProperties.getDriverPruningMultiThreadEnableFilesCount
+    CarbonProperties.getInstance()
+      .addProperty(CarbonCommonConstants.CARBON_MAX_DRIVER_THREADS_FOR_BLOCK_PRUNING, "2")
+    CarbonProperties.getInstance()
+      .addProperty(CarbonCommonConstants.CARBON_DRIVER_PRUNING_MULTI_THREAD_ENABLE_FILES_COUNT, "1")
+    try {
+      sql("CREATE TABLE filtertestTables (ID int, date Timestamp, country String, " +
+        "name String, phonetype String, serialname String, salary int) " +
+        "STORED AS carbondata"
+      )
+      val csvFilePath = s"$resourcesPath/datanullmeasurecol.csv"
+      sql(
+        s"LOAD DATA LOCAL INPATH '" + csvFilePath + "' INTO TABLE " +
```

Comment:
can format these lines. reduce to one line

Reply:
done